docs/DeveloperGuide.adoc

```diff
@@ -789,12 +823,29 @@ Windows, Linux, Unix, OS-X
 ....
 
 [[private-contact-detail]]
-Private contact detail
+
+Private Contact Detail
 
 ....
 A contact detail that is not meant to be shared with others
 ....
 
+[[hidden-contact]]
+
+Hidden Contact
+
+....
+Contacts that require secondary password protection
```

Comment:
Maybe a better description is contacts that can't be found with `find`. To distinguish from private contacts, which shows up with `find`, but details are protected by a password.